app/assets/javascripts/view_data.js

```diff
@@ -70,11 +73,18 @@
         bindto: '#' + $(obj).attr('id'),
         data: {
           columns: [
-            currentData,
-            previousData
+            previousData,
+            currentData
           ],
           regions: cRegions
```

Comment:
This is because we only have useful data for Dec-Jan in the previous time-period.  Not really a legend thing, more that we don't have enough data.  If we had Feb-Jan for the previous time period, it'd look better.

Reply:
👍